toolkit/scss/forms/_option-select.scss

```diff
@@ -131,3 +131,15 @@
     }
   }
 }
+
+.govuk-option-select {
```

Comment:
I agree about the danger but putting it above the copied block seems a bit odd seeing as it being below means the cascade reflects the order of inheritance:

GOVUK option-select component -> Digital Marketplace option-select (in DM Toolkit) -> app-level styles

What about putting a comment above this block to make it clear the separation instead?